src/ui/src/builder/BuilderSettingsVisibility.vue

```diff
@@ -47,9 +47,24 @@
 							setVisibleValue(
 								component.id,
 								(ev.target as HTMLInputElement).value,
+								component.visibleReversed,
 							)
 					"
 				/>
+				<div class="flexRow">
+					<input
+						type="checkbox"
+						:checked="component.visibleReversed"
+						@input="
+							(ev: Event) =>
+								setVisibleValue(
+									component.id,
+									component.visible,
+									(ev.target as HTMLInputElement).checked,
+								)
+						"
```

Comment:
There is a shorter syntax
```suggestion
						@input="setVisibleValue(component.id, component.visible, $event.target.checked)"
```

Reply:
My IDE raises a typing error with this syntax. 

![image](https://github.com/user-attachments/assets/86221b8e-73ca-4f6d-bd71-7991593cb894)